linux/meshd-nl80211.c

```diff
@@ -1303,6 +1306,9 @@ meshd_parse_libconfig (struct config_setting_t *meshd_section,
     CONFIG_LOOKUP(mcast-rate, mcast_rate);
     CONFIG_LOOKUP(beacon-interval,beacon_interval);
 
+    config_setting_lookup_int(meshd_section, "protected-management-frames",
+            (config_int_t *) &config->protected_management_frames);
```

Comment:
wpa_supp also default "ieee80211w" to 0. So perhaps default to 0 is fine here. By the way, encrypting the management frame for kernel 4.8, can you elaborate more.